### DISCUSSION TITLE
Project Inquiry: Cloud-Deployed Chatbot with Advanced Data Interaction

### DISCUSSION THREAD

**Discussion by pacman2410** [FlowiseAI/Flowise#898] [Q&A]
Hello, 
I possess a substantial folder containing numerous PDFs. My objective is to efficiently interact with all the files in order to extract the most pertinent data from them. Following this, I would like the system to either suggest potential action items for me based on the extracted information or facilitate the drafting of emails to clients using the relevant data.

A crucial feature I am seeking is the ability for the chatbot to retain memory of our previous exchanges, ensuring a fluid and intuitive user experience, as well as being able to connect to the internet to bring in real-time data or connect to other websites to pull in data. 

I am considering deploying this solution to the cloud to ensure easy access and scalability. However, I am uncertain about how to conceptualize and structure this project effectively. I would greatly appreciate guidance on how to proceed, especially if it involves utilizing prompts similar to those used in chat "C" to facilitate interactions.

Thank you for your assistance.

---

**Comment by groenewoud** [comment]
Hi,
ping pe and we could chat. It may align with some things, which I am developing.
Best regards,
Paul
